roofit/roofitcore/inc/RooFuncWrapper.h

```diff
@@ -43,6 +50,26 @@ class RooFuncWrapper final : public RooAbsReal {
          throw std::runtime_error(errorMsg.str().c_str());
       }
       _func = (Func)gInterpreter->ProcessLine((funcName + ";").c_str());
+
+      // Calculate gradient
+      gInterpreter->ProcessLine("#include <Math/CladDerivator.h>");
+      // disable clang-format for making the following code unreadable.
+      // clang-format off
+      std::string requestFunc = "#pragma clad ON\n"
+                                 "void " + requestName + "() {\n"
+                                 "  clad::gradient(" + funcName + ");\n"
+                                 "}\n"
+                                 "#pragma clad OFF";
+      // clang-format on
+      comp = gInterpreter->Declare(requestFunc.c_str());
+      if (!comp) {
```

Comment:
```suggestion
      if (!gInterpreter->Declare(requestFunc.c_str())) {
```
Since you don't use `comp` anywhere else, it's better to do this in the `if` so it's clear the result is not used anywhere else (also the same a few lines above).

Reply:
Ah, I would prefer to keep it this way because otherwise, it becomes a bit confusing to see where the function is declared haha...

Reply:
Okay! That is also a fair point